Refit/VV-Refit.csproj

```diff
@@ -2,7 +2,9 @@
 
   <PropertyGroup>
     <Product>Refit ($(TargetFramework))</Product>
-    <TargetFrameworks>netstandard1.4;net45</TargetFrameworks>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+	  <PackageVersion>1.0.0</PackageVersion>
```

Comment:
No need to add package version as it is patched by the build

Reply:
Please remove that i made another PR. #414

Reply:
Due some mistakes, I made another pull request with all fixed. #414